Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -222,8 +222,9 @@ - (BOOL)prefersStatusBarHidden
 
     m_videoDimensions = videoDimensions;
     ALWAYS_LOG_IF_POSSIBLE(LOGIDENTIFIER, videoDimensions);
-    for (auto& client : copyToVector(m_clients))
-        client->videoDimensionsChanged(videoDimensions);
+    m_clients.forEach([&](auto& client) {
```

Comment:
Is it safe to remove the "copy to vector" technique in all these places? Maybe the weak container always does that. If so it makes me wonder if the `forEach` function name needs a way to make that explicit since I'm sure not all `forEach` functions would take such a precaution about the collection being modified while iterating.

Reply:
Yes, the `forEach()` function was added exactly for this purpose. It does copy the elements to a Vector first. It's true that ideally the name should convey that. I'll try and come up with a better name.

Reply:
Coming up with a good name is not easy here. I can follow-up if someone find a name we like better.

I kinda like `forEach` because it is short. It doesn't convey that it makes a copy first but it makes code safe by default which matches our new way of doing things. If someone needs a version that doesn't do the copy, they can always add it and call it `unsafeForEach()` :)